trie-db/src/iterator.rs

```diff
@@ -447,6 +489,104 @@ impl<L: TrieLayout> TrieDBRawIterator<L> {
 		}
 	}
 
+	fn next_back_raw_item(
```

Comment:
if using same trail and crumb then we can avoid this duplicated method (just pass direction in parameter to `next_raw_item` function and use either increment or decrement depending on it. 
Actually we also could not have `increment` and `decrement` function but just `increment` with a parameter `direction`: would avoid lot of conditional here.